dvc/utils/compat.py

```diff
@@ -46,6 +45,8 @@ def _makedirs(name, mode=0o777, exist_ok=False):
         except OSError as e:
             if e.errno == errno.EEXIST:
```

Comment:
``` 
           if e.errno == errno.EEXIST:
                pass	           
            else:
                raise e
```

can be replaced with

```
            if e.errno != errno.EEXIST:
                raise
```

Reply:
Totally! Just submitted a commit with that fix :)